src/styles/_header.scss

```diff
@@ -0,0 +1,54 @@
+.navbar {
```

Comment:
looks good. The only issue is the filename. Unless there is a reason you have an underscore prepended on the filename, rename the file from `_header.scss` to `header.scss`